### ISSUE TITLE
no PPA named 'ubuntu/byzanz'

### ISSUE THREAD

**Issue by tasdikrahman** [IonicaBizau/gif-recorder#1]
I am on `Ubuntu 14.10` and I tried adding the ppa which you have mentioned `sudo add-apt-repository ppa:fossfreedom/byzanz`

But doing that I am getting an error message saying `the ppa named ubuntu/byzanz' does not exist

``` bash
tasdik@Acer:~$ sudo -E add-apt-repository ppa:fossfreedom/byzanz
[sudo] password for tasdik: 
 screencast tool for gnome-classic/ unity-2D and unity 3D.  

See this Q&A for more details: http://askubuntu.com/a/123515/14356

See this Q&A for a nice GUI script: http://askubuntu.com/a/201018/14356
 More info: https://launchpad.net/~fossfreedom/+archive/ubuntu/byzanz
Press [ENTER] to continue or ctrl-c to cancel adding it


Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.4/urllib/request.py", line 1174, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/lib/python3.4/http/client.py", line 1090, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python3.4/http/client.py", line 1128, in _send_request
    self.endheaders(body)
  File "/usr/lib/python3.4/http/client.py", line 1086, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python3.4/http/client.py", line 924, in _send_output
    self.send(msg)
  File "/usr/lib/python3.4/http/client.py", line 859, in send
    self.connect()
  File "/usr/lib/python3.4/http/client.py", line 1230, in connect
    server_hostname=sni_hostname)
  File "/usr/lib/python3.4/ssl.py", line 364, in wrap_socket
    _context=self)
  File "/usr/lib/python3.4/ssl.py", line 577, in __init__
    self.do_handshake()
  File "/usr/lib/python3.4/ssl.py", line 804, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:600)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 101, in _get_https_content_py3
    lp_page = urllib.request.urlopen(request, cafile=LAUNCHPAD_PPA_CERT)
  File "/usr/lib/python3.4/urllib/request.py", line 153, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.4/urllib/request.py", line 455, in open
    response = self._open(req, data)
  File "/usr/lib/python3.4/urllib/request.py", line 473, in _open
    '_open', req)
  File "/usr/lib/python3.4/urllib/request.py", line 433, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.4/urllib/request.py", line 1217, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/usr/lib/python3.4/urllib/request.py", line 1176, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error EOF occurred in violation of protocol (_ssl.c:600)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 321, in get_ppa_info
    ret = get_ppa_info_from_lp(user, ppa)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 91, in get_ppa_info_from_lp
    return get_info_from_lp(lp_url)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 87, in get_info_from_lp
    return _get_https_content_py3(lp_url)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 107, in _get_https_content_py3
    raise PPAException("Error reading %s: %s" % (lp_url, reason), e)
softwareproperties.ppa.PPAException: 'Error reading https://launchpad.net/api/1.0/~fossfreedom/+archive/ubuntu/byzanz: EOF occurred in violation of protocol (_ssl.c:600)'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.4/threading.py", line 868, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py", line 688, in addkey_func
    func(**kwargs)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 376, in add_key
    return apsk.add_ppa_signing_key()
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 234, in add_ppa_signing_key
    ppa_info = get_ppa_info(ppa_path)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 333, in get_ppa_info
    _get_suggested_ppa_message(user, ppa))
softwareproperties.shortcuts.ShortcutException: Cannot add PPA: 'ppa:~fossfreedom/ubuntu/byzanz'.
The user named '~fossfreedom' has no PPA named 'ubuntu/byzanz'
Please choose from the following available PPAs:
 * 'byzanz':  byzanz screencaster
 * 'g2ipmsg':  G2ipmsg
 * 'indicator-sysmonitor':  System Monitor Application Indicator
 * 'libmtp':  libmtp
 * 'nautilus':  Nautilus with bookmark icons
 * 'packagefixes':  Package Management Fixes
 * 'playground':  playground
 * 'rhythmbox':  Rhythmbox
 * 'rhythmbox-plugins':  Rhythmbox third-party plugins
 * 'testbuilds':  Test builds
 * 'xfwm4':  XFCE Tiling Manager

tasdik@Acer:~$ 
```

any help on that?

**Comment by fossfreedom** [comment]
you no longer have to install byzanz via my PPA.  From 14.04 onwards, byzanz is available in the universe repository - just make sure that this repository is enabled via your software sources (software center)

**Comment by tasdikrahman** [comment]
I installed it using through `apt-get`. Thanks anyway :+1:

**Comment by IonicaBizau** [comment]
@prodicus Thanks for heads-up!
@fossfreedom Thanks for info. Docs should be updated. :memo:

**Comment by tasdikrahman** [comment]
@IonicaBizau No problem!

**Comment by IonicaBizau** [comment]
We will close this when it's fixed. :sparkle:

**Comment by tasdikrahman** [comment]
@IonicaBizau Sorry about that :)

**Comment by IonicaBizau** [comment]
Fixed via #2. :fire: 

:ship: **`1.0.0`**
